monai/apps/detection/transforms/array.py

```diff
@@ -13,12 +13,19 @@
 https://github.com/Project-MONAI/MONAI/wiki/MONAI_Design
 """
 
-from typing import Type, Union
+from typing import Sequence, Type, Union
+
+import numpy as np
+import torch
 
 from monai.config.type_definitions import NdarrayOrTensor
-from monai.data.box_utils import BoxMode, convert_box_mode, convert_box_to_standard_mode
+from monai.data.box_utils import BoxMode, convert_box_mode, convert_box_to_standard_mode, get_spatial_dims
 from monai.transforms.transform import Transform
+from monai.utils import ensure_tuple, ensure_tuple_rep, fall_back_tuple, look_up_option
 from monai.utils.enums import TransformBackends
+from monai.utils.type_conversion import convert_data_type
+
+from .box_ops import apply_affine_to_boxes, resize_boxes, zoom_boxes
 
 __all__ = ["ConvertBoxToStandardMode", "ConvertBoxMode"]
```

Comment:
Could you please help add the new transforms in the `all` list?

Thanks.

Reply:
Oh, I forgot. Thanks